<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog1</class>
 <widget class="QDialog" name="Dialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>177</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="savebtn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Century Gothic&quot;;
background-color: rgb(108, 108, 108);
color: rgb(255, 255, 255);
padding: 0 2%;
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>Enregistrer</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/image/image/valider.png</normaloff>:/image/image/valider.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Century Gothic&quot;;
background-color: rgb(108, 108, 108);
color: rgb(255, 255, 255);
padding: 0 2%;
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/image/image/annuler.png</normaloff>:/image/image/annuler.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Century Gothic</family>
         </font>
        </property>
        <property name="text">
         <string>Nom du Fichier       :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="projectName">
        <property name="font">
         <font>
          <family>Century Gothic</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Century Gothic</family>
         </font>
        </property>
        <property name="text">
         <string>Emplacement        :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="chemin">
        <property name="font">
         <font>
          <family>Century Gothic</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Browse">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Century Gothic&quot;;
background-color: rgb(108, 108, 108);
color: rgb(255, 255, 255);
padding: 0 2%;
border-radius: 5px;</string>
        </property>
        <property name="text">
         <string>Parcourir...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Century Gothic</family>
         </font>
        </property>
        <property name="text">
         <string>Fichier Train             :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ficTrain">
        <property name="font">
         <font>
          <family>Century Gothic</family>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Century Gothic</family>
         </font>
        </property>
        <property name="text">
         <string>Fichier Place           :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ficPlace">
        <property name="font">
         <font>
          <family>Century Gothic</family>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Century Gothic</family>
         </font>
        </property>
        <property name="text">
         <string>Fichier Itinéraire      :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ficItineraire">
        <property name="font">
         <font>
          <family>Century Gothic</family>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Century Gothic</family>
         </font>
        </property>
        <property name="text">
         <string>Fichier Réservation:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ficReservation">
        <property name="font">
         <font>
          <family>Century Gothic</family>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="ficRecette">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>268</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
